code/game/machinery/excelsior/blackshield_teleporter.dm

```diff
@@ -64,7 +64,7 @@ var/global/blackshield_max_energy //Maximaum combined energy of all teleporters
 		/obj/item/gun/projectile/automatic/duty = 325,
 		/obj/item/gun/projectile/automatic/sts/rifle = 400,
 		/obj/item/gun/projectile/automatic/drozd/NM_colony = 375,
-		/obj/item/gun/projectile/automatic/ppsh/NM_colony = 300,
```

Comment:
did you intend to remove the ppsh?

Reply:
The ppsh is a starting weapon. I wanted to reduce bloat by not having more options than needed.